app/assets/javascripts/admin/addon/components/schema-theme-setting/field.gjs

```diff
@@ -52,14 +52,9 @@ export default class SchemaThemeSettingField extends Component {
           @value={{@value}}
           @spec={{@spec}}
           @onChange={{@onValueChange}}
+          @description={{this.description}}
         />
       </div>
-
-      {{#if this.hasDescription}}
-        <div class="schema-field__description">
-          {{htmlSafe this.description}}
-        </div>
-      {{/if}}
```

Comment:
I need to move this out of this component into the individual field type components because the way we show/hide description can and may vary between input types.